transmartApp/grails-app/services/com/recomdata/transmart/data/export/ExportMetadataService.groovy

```diff
@@ -102,7 +103,39 @@ class ExportMetadataService {
 
         hdMetaData
     }
-
+    
+    def getHighDimMetaData(OntologyTerm term) {
+        
+        // Retrieve all descendant terms that have the HIGH_DIMENSIONAL attribute
+        // def terms = term.getAllDescendants() + term
```

Comment:
Are those commented codes still used? If not pls remove, if yes pls add TODO about what is it for.